lightkurve/correctors/tests/test_cbvcorrector.py

```diff
@@ -0,0 +1,53 @@
+""" cbvcorrector.py module unit tests
+"""
+
+import pytest
+from numpy.testing import (assert_almost_equal, assert_array_equal,
+                           assert_allclose, assert_raises)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .. import get_cbvs, CotrendingBasisVectors, KeplerCotrendingBasisVectors
+
+def test_cbv_nonretrieval():
+    """ Tests that do not require remote_data
+    """
+
+    # Create a simple CotrendingBasisVectors object
+    desired_indices = [1,2,4,6,8]
+    cbvs = CotrendingBasisVectors('Kepler', cbv_type='SingleScale',
+            cbv_indices=desired_indices)
+    assert cbvs.mission == 'Kepler'
+    assert cbvs.cbv_type == 'SingleScale'
+    assert cbvs.cbv_indices == desired_indices
+    assert cbvs.band is None
+
+    # These should fail
+    assert_raises(ValueError, CotrendingBasisVectors('SuperKepler', cbv_type='SingleScale',
+            cbv_indices=desired_indices))
+  # assert_raises(ValueError, CotrendingBasisVectors('Kepler', cbv_type='SuperSingleScale',
+  #         cbv_indices=desired_indices))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [indentation is not a multiple of four (comment)](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4896575)